packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,18 @@
+If we're not working with arrays, we can also create our own array of
+templates to render, for example, by using a `for...of` loop.
+
+```ts
+const templates = [];
+for (const item of this.items) {
+  templates.push(html`<li>${item}</li>`);
+}
+```
+
+It can then be placed inside the template returned from the render method.
+```ts
+return html`
+<ul>
+  ${templates}
+</ul>
+`
+```
```

Comment:
perhaps lead into the next step a bit. something like "it works but is uggo code"